Tools/CISupport/ews-build/steps.py

```diff
@@ -80,6 +80,12 @@ class GitHub(object):
     def repository_urls(cls):
         return [GITHUB_URL + project for project in GITHUB_PROJECTS]
 
+    @classmethod
+    def user_for_queue(cls, queue):
+        if queue.lower() in ['unsafe-merge-queue', 'merge-queue']:
```

Comment:
Should we add commit-queue here?

Reply:
I suppose we will need GitHub credentials on this bot in the very near future.